environments/icds-new/inventory.ini

```diff
@@ -392,6 +392,183 @@ postgresql_checkpoint_completion_target=0.9
 postgresql_wal_buffers=16MB
 postgresql_default_statistics_target = 100
 
+
+[pgshard1standby]
+10.247.164.5
+
+[pgshard1standby:vars]
+hostname='pgshard1standby'
+devices=["/dev/sdc"]
+partitions=["/dev/sdc1"]
+datavol_device='/dev/mapper/consolidated-data'
+hot_standby_master='10.247.164.20'
+replication_slot='standby'
+pgbouncer_max_connections=3500
+pgbouncer_default_pool=775
+postgresql_max_connections=800
+postgresql_work_mem=41943kB
+postgresql_shared_buffers=8GB
+postgresql_effective_cache_size=24GB
+postgresql_maintenance_work_mem = 2GB
+postgresql_checkpoint_segments=64
+postgresql_checkpoint_completion_target=0.9
+postgresql_wal_buffers=16MB
+postgresql_default_statistics_target = 100
+
+[pgshard10standby]
+10.247.164.12
+
+[pgshard10standby:vars]
+hostname='pgshard10standby'
+devices=["/dev/sdc"]
+partitions=["/dev/sdc1"]
+datavol_device='/dev/mapper/consolidated-data'
+hot_standby_master='10.247.164.77'
+replication_slot='standby'
+pgbouncer_max_connections=3500
+pgbouncer_default_pool=775
+postgresql_max_connections=800
+postgresql_work_mem=41943kB
+postgresql_shared_buffers=8GB
+postgresql_effective_cache_size=24GB
+postgresql_maintenance_work_mem = 2GB
+postgresql_checkpoint_segments=64
+postgresql_checkpoint_completion_target=0.9
+postgresql_wal_buffers=16MB
+postgresql_default_statistics_target = 100
+
+[pgshard11standby]
+10.247.164.21
+
+[pgshard11standby:vars]
+hostname='pgshard11standby'
+devices=["/dev/sdc"]
+partitions=["/dev/sdc1"]
+datavol_device='/dev/mapper/consolidated-data'
+hot_standby_master='10.247.164.78'
+replication_slot='standby'
+pgbouncer_max_connections=3500
+pgbouncer_default_pool=775
+postgresql_max_connections=800
+postgresql_work_mem=41943kB
+postgresql_shared_buffers=8GB
+postgresql_effective_cache_size=24GB
+postgresql_maintenance_work_mem = 2GB
+postgresql_checkpoint_segments=64
+postgresql_checkpoint_completion_target=0.9
+postgresql_wal_buffers=16MB
+postgresql_default_statistics_target = 100
+
+[pgshard12standby]
+10.247.164.22
+
+[pgshard12standby:vars]
+hostname='pgshard12standby'
+devices=["/dev/sdc"]
+partitions=["/dev/sdc1"]
+datavol_device='/dev/mapper/consolidated-data'
+hot_standby_master='10.247.164.79'
+replication_slot='standby'
+pgbouncer_max_connections=3500
+pgbouncer_default_pool=775
+postgresql_max_connections=800
+postgresql_work_mem=41943kB
+postgresql_shared_buffers=8GB
+postgresql_effective_cache_size=24GB
+postgresql_maintenance_work_mem = 2GB
+postgresql_checkpoint_segments=64
+postgresql_checkpoint_completion_target=0.9
+postgresql_wal_buffers=16MB
+postgresql_default_statistics_target = 100
+
+[pgshard5standby]
+10.247.164.23
+
+[pgshard5standby:vars]
+hostname='pgshard5standby'
+devices=["/dev/sdc"]
+partitions=["/dev/sdc1"]
+datavol_device='/dev/mapper/consolidated-data'
+#hot_standby_master='10.247.164.66'
```

Comment:
is this supposed ot be commented out?

Reply:
Oh. Thanks for pointing out. I will correct this.